data/category/base/data.json

```diff
@@ -1,25 +1,52 @@
 {
-    "data": {
-        "category-dairy": {
-            "name": "Dairy"
-        },
-        "category-baking" : {
-            "name": "Baking"
-        },
-        "category-produce" : {
-            "name": "Produce"
-        },
-        "category-bakery" : {
-            "name": "Bakery"
-        },
-        "category-deli" : {
-            "name": "Deli"
-        },
-        "category-frozen" : {
-            "name": "Frozen"
-        },
-        "category-canned" : {
-            "name": "Canned"
-        }
+  "data": {
+    "category-dairy": {
+      "name": "Dairy"
+    },
+    "category-baking": {
+      "name": "Baking"
+    },
+    "category-produce": {
+      "name": "Produce"
+    },
+    "category-bakery": {
+      "name": "Bakery"
+    },
+    "category-deli": {
+      "name": "Deli"
+    },
+    "category-frozen": {
+      "name": "Frozen"
+    },
+    "category-canned": {
+      "name": "Canned"
+    },
+    "category-fish": {
+      "name": "Fish"
+    },
+    "category-meat": {
```

Comment:
Deli is what we call it around here. Similar for you?

Reply:
I tight deli was ready made stuff? We don't really have deli as a category in Germany, can you explain what you usually find there?

Reply:
Oh and we have meat but that's split into meat directly and meat that is put on bread which we call 'wurst' which would translate to sausage (does not make sense this way)

Reply:
You're right. Our meat areas and deli are always right next to each other >.>

Reply:
I like having two categories for meat and deli because I sometimes buy meat at the butcher's. Technically deli could also be cole slaw, couldn't it? I couldn't get that at the butcher's.
Although I guess I would categorize salami as a deli food and I could get that.
Man this is hard....

Reply:
This is another case where I would suggest setting specific foods to be set to buy from specific stores rather than using category to determine where you should be going to shop.

Reply:
ok so i vote for splitting meat (actually cuts of meant meant for meals) and deli (if i understand correctly any kind of prepared food be it cooked meat, salads or whatever) and there also needs to be one category for meat thats meant to be put on bread, not sure what that would be called in english.

I also say we add a field "description" to the category so that people needing to decide where to add something have some reference.

Reply:
A description field would be great. We could add the field here and just use it for our documenting purppses until we get it implemented in the app later, right?